docs/input-manifest/api-reference.md

```diff
@@ -29,167 +29,139 @@ Manifest is a definition of the user's infrastructure. It contains cloud provide
 Contains configurations for supported cloud providers. At least one provider
 needs to be defined.
 
-- `gcp` [GCP](#gcp)
-  
-  List of configuration options for [Google Cloud](https://cloud.google.com/). This field is optional.
+- `name`
 
-- `hetzner` [Hetzner](#hetzner)
-  
-  List of configuration options for [Hetzner Cloud](https://www.hetzner.com/cloud) . This field is optional.
+  The name of the provider specyfication. It has to be unique across all providers.
```

Comment:
`specyfication` -> `specification`

Also, I think we call them `provider instances`, so maybe `The name of the provider instance which has to be unique across all providers.`